impeller/renderer/backend/gles/buffer_bindings_gles_unittests.cc

```diff
@@ -33,14 +34,11 @@ TEST(BufferBindingsGLESTest, BindUniformData) {
   DeviceBufferGLES device_buffer(DeviceBufferDescriptor{.size = sizeof(float)},
                                  reactor, backing_store);
   BufferView buffer_view(&device_buffer, Range(0, sizeof(float)));
-  vertex_bindings.buffers.push_back(BufferAndUniformSlot{
-      .slot =
-          ShaderUniformSlot{
-              .name = "foobar", .ext_res_0 = 0, .set = 0, .binding = 0},
+  bound_buffers.push_back(BufferAndUniformSlot{
       .view = BufferResource(&shader_metadata, buffer_view)});
-  Bindings fragment_bindings;
-  EXPECT_TRUE(bindings.BindUniformData(mock_gl->GetProcTable(), vertex_bindings,
-                                       fragment_bindings));
+
+  EXPECT_TRUE(bindings.BindUniformData(mock_gl->GetProcTable(), bound_textures,
+                                       bound_buffers, 0, 0, 0, 1));
```

Comment:
its not going to be garbage data exactly, just a default constructed struct with values of 0 and a null name. at any rate, Its easy to remove so I'll just do that.

Reply:
I might just be bad at C++, but I though for types with a natural zero it was pretty well defined that it would be initialized to 0, provided that we aren't doing something weird like reinterpret_casting random bytes.

Anyway, slot data is removed.

Reply:
I think you may be right, this is a special case for using aggregate initialization (`struct = {...}` and not specifying all the fields.  I would have hoped our linter would complain if it was garbage.